.travis.yml

```diff
@@ -48,6 +48,8 @@ matrix:
       env: SYMFONY=3.3.*
     - php: '7.1'
       env: SYMFONY='dev-master as 3.3'
+    - php: 7.1
+      env: DEPENDENCIES=beta
```

Comment:
We manage this file through dev-kit, you aren't supposed to touch this.

Reply:
I saw that in the note in this file. @dbu suggested it would be okay to edit this to start off with, to see what breaks, before finally updating it via the dev-kit.

Reply:
No need to, look at the Travis jobs, there is one for sf 4 (dev-master as 3.3)

Reply:
I'll just remove this then :)